lib/_stream_transform.js

```diff
@@ -202,6 +203,32 @@ Transform.prototype._destroy = function(err, cb) {
   });
 };
 
+Transform.prototype.acquireStandardStream = function() {
+  emitExperimentalWarning('Transform.acquireStandardStream');
+  return new TransformStream({
+    start: (controller) => {
+      this.on('data', (chunk) => {
+        controller.enqueue(chunk);
+      });
+      this.on('end', () => controller.close());
+      this.once('error', (e) => controller.error(e));
+    },
+    transform: (chunk) => {
+      return new Promise((resolve) => {
+        const underHighWaterMark = this.write(chunk);
+        if (!underHighWaterMark) {
+          this.once('drain', resolve);
+        } else {
+          resolve();
+        }
+      });
+    },
+    flush: () => {
+      this.destroy();
```

Comment:
I'm not certain this is correct.

Reply:
I would think `end()` would be more correct here? That would seem to be a closer match to the definition of `flush()`